aspnetcore/security/enforcing-ssl.md

```diff
@@ -345,6 +345,8 @@ The following command provides help on the `dev-certs` tool:
 dotnet dev-certs https --help
 ```
 
+Be aware of the environment in which the ASP.NET Core development certificate is being installed. It's important to avoid having it installed in an environment that will be redistributed, such as a container image or virtual machine. Redistribution of the certificate's private key risks it being maliciously used to spoof the identity of a victim. In order to prevent the certificate from being generated during the .NET CLI's first-run experience, set the `DOTNET_GENERATE_ASPNET_CERTIFICATE` environment variable to `false`.
```

Comment:
Be stronger. Something like.

_You must not create a development certificate in an environment that will be redistributed, such as a container image or virtual machine. Such a use to spoofing and elevation of privilege. To prevent this  set the `DOTNET_GENERATE_ASPNET_CERTIFICATE` environment variable to `false`._